app/src/main/java/com/sq/thed_ck_licker/ecs/systems/cardSystems/CardCreationSystem.kt

```diff
@@ -53,7 +50,8 @@ class CardCreationSystem private constructor(@Suppress("unused") private val com
 
     fun addDamageCards(amount: Int): List<EntityId> {
```

Comment:
This one could be separate card like KillDamageTestCard or TakeTonsOfDamageTestCard or something like that.
Tho this is fine too.

Reply:
Yep yep, tho the cards will be redone multiple times so this is fine too for now.